src/Appwrite/Platform/Modules/Sites/Http/Sites/Create.php

```diff
@@ -76,7 +76,7 @@ public function __construct()
             ->param('outputDirectory', '', new Text(8192, 0), 'Output Directory for site.', true)
             ->param('subdomain', '', new CustomId(), 'Unique custom sub-domain. Valid chars are a-z, A-Z, 0-9, period, hyphen, and underscore. Can\'t start with a special char. Max length is 36 chars.', true)
             ->param('buildRuntime', '', new WhiteList(array_keys(Config::getParam('runtimes')), true), 'Runtime to use during build step.')
-            ->param('adapter', '', new Text(8192, 0), 'Framework adapter. Allows: static, ssr', true)
+            ->param('adapter', '', new WhiteList(['static', 'ssr']), 'Framework adapter defining rendering strategy. Allowed values are: static, ssr', true)
```

Comment:
We call it `ssg` in detection library. Do we wish to call it `static` or `ssg` here?

Reply:
I recall discussions calling it something else than adapter too.

While we call it adapter, static and ssr is accurate.

Once we rename it to strategy/rendering, then we should replace whitelist too